inverter/signal_plotter.py

```diff
@@ -120,7 +120,7 @@ def main(self):
             axes[1,1].set_xlim(0,(nsamp-1)/self.Rs)
             axes[1,1].grid(True)
         else:
-            if self.plotmodel == 'sv' or self.plotmodel == 'vhdl':
+            if self.plotmodel in [ 'placeholder' ]:
```

Comment:
@mkosunen what is this?

Reply:
Previously different models had different latencies that I used to align the singnals. After I changed the sampling of Z to falling edge, latencies are the same, thus the placeholder is for the model that would need a different latency. 

Thi scould be simplified. I just left it there.